src/tools/ceph-dencoder/rbd_types.h

```diff
@@ -19,14 +19,6 @@ TYPE(rbd_replay::action::ActionEntry)
 TYPE(rbd::mirror::image_map::PolicyData)
 #endif
 
-#if defined(WITH_RBD) && defined(WITH_RBD_SSD_CACHE)
-#include "librbd/cache/pwl/Types.h"
-#include "librbd/cache/pwl/SSDTypes.h"
-TYPE(librbd::cache::pwl::WriteLogPmemEntry)
```

Comment:
Nit: why did you remove these?

Reply:
As the PWL is made as a standalone plugin, do you think it is OK to include this PWL related types in ceph_dencoder?

Reply:
`ceph_dencoder` is a test package that helps to ensure that we don't accidentally break backwards compatibility.

Reply:
Added it back. Thanks.